twins/mongodb/twins.go

```diff
@@ -23,15 +23,13 @@ type twinRepository struct {
 
 var _ twins.TwinRepository = (*twinRepository)(nil)
 
-// NewTwinRepository instantiates a MongoDB implementation of twin
-// repository.
+// NewTwinRepository instantiates a MongoDB implementation of twin repository.
 func NewTwinRepository(db *mongo.Database) twins.TwinRepository {
 	return &twinRepository{
 		db: db,
 	}
 }
 
-// Save persists the twin
```

Comment:
All exported functions must have comments and linter should warn you.

So I think that you should not remove this and fix your linter in your editor so that it warns you about things like this.

Reply:
Looks that this is not needed when functions are implementation of an interface.